src/script/lib/config.ts

```diff
@@ -54,7 +54,8 @@ export default class Config {
     filterMethod: 0, // ['Censor', 'Substitute', 'Remove'];
     filterWordList: true,
     globalMatchMethod: 3, // ['Exact', 'Partial', 'Whole', 'Per-Word', 'RegExp']
-    muteAudio: false,
+    muteAudio: false, // Filter audio
+    filterAudioMethod: 1, // 0: Mute Tab, 1: Video Volume
```

Comment:
I'm not familiar with the code base, but I'm assuming this is configurable somehow.

Reply:
It will be! I forgot to mark this as WIP.